src/cpp/library/cstdlib

```diff
@@ -110,7 +110,7 @@ unsigned long int strtoul(const char * str, char ** endptr, int base);
 
 char * getenv(const char * name) {
 	size_t size = strlen(name) + 1;
-	char tmp[size];
+	char* tmp = (char*)malloc(size);
```

Comment:
Is this a fix for the warning message below? 
```
./cstdlib:115:9: warning: address of stack memory associated with local variable 'tmp' returned [-Wreturn-stack-address]
        return tmp;
```

If so, I don't think we actually need a fix here for the following reasons: 
1. There's no lexical scopes in GOTO functions and destruction of local variable is not enforce on function exit. The existing GOTO functions look quite right: 
```cpp
// GOTO functions - a simplified version of getenv and its usage in main function

getenv (c:@F@getenv#*1C#):
  unsigned long int size;
  size= 1;
  tmp$2=size;
  signed char [tmp$2] tmp;
        
  DYNAMIC_SIZE(&tmp)=(unsigned int)(tmp$2 * 1);
  FUNCTION_CALL:  strcpy(&tmp[0], name)
  
  RETURN: &tmp[0]
  
  END_FUNCTION // getenv
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

main (c:@F@main#):
  signed char * env_p;

  FUNCTION_CALL:  return_value$_getenv$1=getenv(&"PATH"[0])
  env_p=return_value$_getenv$1;

  RETURN: 0
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```
 
2. This fix completely changed the GOTO behaviour in ESBMC backend - `[]` is just array subscript operator verification (e.g. boundary check stuff), but `malloc` requires memory management verification actions (e.g. use-after-free stuff).